ci/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM fedora
+MAINTAINER Igor Gnatenko <i.gnatenko.brain@gmail.com>
+
+WORKDIR /opt/rpm
+COPY . .
+
+RUN echo -e "deltarpm=0\ninstall_weak_deps=0\ntsflags=nodocs" >> /etc/dnf/dnf.conf
+RUN dnf -y update
+RUN dnf -y install \
```

Comment:
well, yes and no. 

* If RPM starts/stops requiring something -- it is only way how to properly deal with that (especially with RPM release cadence)
* We don't really need all deps for all tests here